### ISSUE TITLE
componentWillMount deprecated in React 16.9

### ISSUE THREAD

**Issue by Juholei** [reagent-project/reagent#445]
As detailed earlier in https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html, some React lifecycle methods will be deprecated in the future. With  React 16.9, the deprecation warning is now in place ([release notes](https://github.com/facebook/react/blob/master/CHANGELOG.md#1690-august-8-2019)). Using Reagent with React 16.9 produces these warnings in the console:
```
Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move code with side effects to componentDidMount, and set initial state in the constructor.
* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: ReagentInput, <insert every Reagent component defined here>
```

**Comment by Juholei** [comment]
I checked the issues before posting this to see if it was already reported, but now I see instantly afterwards there's already a pull request addressing this 🤦‍♂ - sorry for the unnecessary issue.

**Comment by Deraen** [comment]
Fixed in 0.9.
